subsys/bluetooth/host/att.c

```diff
@@ -2441,7 +2441,8 @@ static int bt_att_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
 
 	if (!handler) {
 		BT_WARN("Unhandled ATT code 0x%02x", hdr->code);
-		if (att_op_get_type(hdr->code) != ATT_COMMAND) {
+		if ((att_op_get_type(hdr->code) != ATT_COMMAND) &&
+		    (att_op_get_type(hdr->code) != ATT_INDICATION)) {
```

Comment:
Minor: the () around `(a != b)` isn't really necessary, and in fact inconsistent with style used in other places in Bluetooth code. Take a look at e.g. line 2456/2457 below.

Reply:
Changed